lib/matplotlib/mlab.py

```diff
@@ -186,6 +186,8 @@
     long = int
 
 
+@cbook.deprecated("2.1",
+                  alternative='np.logspace(np.log(xmin), np.log(xmax), N)')
```

Comment:
or geomspace

Reply:
What? Why? Where is 'geomspace'?  The alternative looks fine to me as-is.

Reply:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.geomspace.html
TBH I don't care enough about mlab to get into a debate of whether we should recommend an old function with a different behavior, or a new function with the same behavior.

Reply:
So new I wasn't aware of it. Thank you.  I guess I would say "np.logspace or np.geomspace (numpy >= 1.13)" and leave the rest to the user to figure out.

Reply:
@dstansby  For the record (more thqn for qnother reason as the current suggestions are to avoid specifying the arguments), it should be `np.logspace(np.log10(xmin), np.log10(xmax), N)` (the default base for `np.logspace` is 10).